packages/ceramic-docid/README.md

```diff
@@ -0,0 +1,13 @@
+# Ceramic DocId
+
+> This package contains Ceramic DocId implementation.
+
+### Installation
+```
+$ npm install @ceramicnetwork/ceramic-docid
```

Comment:
Can you change the package and folder name to just be `docid` we are changing this for all packages as well later.
So the package would be `@ceramicnetwork/docid`

Reply:
yup, this makes more sense